src/test/run-make/issue-11908/Makefile

```diff
@@ -13,11 +13,11 @@ all:
 	mv $(call DYLIB,foo) $(TMPDIR)/other
 	$(RUSTC) foo.rs --crate-type=dylib
 	$(RUSTC) bar.rs -L $(TMPDIR)/other 2>&1 | \
-		grep "multiple dylib candidates"
+		grep "assertion failed" && exit 1 || exit 0
```

Comment:
What are the "strange errors" that get printed here? 

"The compiler doesn't assert" is such a broad test case it's essentially not testing anything unfortunately.